.github/ISSUE_TEMPLATE.md

```diff
@@ -3,29 +3,29 @@
 2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://Rook-io.slack.com).
 3. Did you already search the existing open issues for anything similar? -->
 
-Is this a bug report or feature request?
+**Is this a bug report or feature request?**
 <!-- Remove only one -->
 * Bug Report
 * Feature Request
 
 **Bug Report**
 
-What happened:
+**Deviation from expected behavior:**
 
-What you expected to happen:
+**Expected behavior:**
```

Comment:
This is a nit, but I'd switch the location of this so that it comes before the deviation portion.